documenteer/bin/refreshlsstbib.py

```diff
@@ -0,0 +1,119 @@
+"""Command line executable to refresh lsst bib files.
+"""
+
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+from builtins import *  # NOQA
```

Comment:
I've been doing this as a boilerplate to avoid surprises. Maybe you're right that it's an anti-pattern.